modules/apps/object/object-test/src/testFunctional/paths/CreateObject.path

```diff
@@ -129,6 +129,11 @@
 		<td>VIEW_ENTRY_SELECTED_ON_RELATIONSHIP_FIELD</td>
 		<td>//input[contains(@name,'r_relationship') and contains(@value,'${key_entry}')]</td>
 		<td></td>
+	</tr>
+		<tr>
+		<td>VIEW_ENTRY_RELATIONSHIP_FIELD</td>
+		<td>//div[contains(@class,'dnd-td')][3]//span[contains(text(),'${key_entry}')]</td>
+		<td></td>
```

Comment:
This is not formatted correctly. Also if you're using an index like [3] there is a high chance this path is not reusable. Please look for an existing path or create one that can be reused.